upgrade

```diff
@@ -93,6 +100,18 @@ main() {
         docker load -i "$COOG_IMAGE_ARCHIVE" || return $?
         COOG_IMAGE_TAG="$(docker_last_image)"
     fi
+    if [ "$COOG_IMAGE_REPO" != "-" ]
+    then
+        if [ ! -f "$HOME/.docker/config.json" ]
```

Comment:
no need to check => docker pull directly and docker will print the right message
We can have an empty file @ ~/.docker/config.json...